sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -26,6 +26,10 @@
                           pull_frequency_hours=_PULL_FREQUENCY_HOURS)
 _image_df = common.read_catalog('gcp/images.csv',
                                 pull_frequency_hours=_PULL_FREQUENCY_HOURS)
+if _image_df[_image_df['Tag'] == 'skypilot:cpu-ubuntu-2004'].empty:
+    # Update the image catalog if it does not include the updated images
+    # https://github.com/skypilot-org/skypilot-catalog/pull/25.
+    _image_df = common.read_catalog('gcp/images.csv', pull_frequency_hours=0)
```

Comment:
Just to make sure, after increasing our version number, we don't need this right?